theme_bewise/views/customizations.xml

```diff
@@ -82,136 +119,356 @@
     <xpath expr="//section" position="attributes">
         <attribute name="data-oe-shape-data">{"shape":"web_editor/Blobs/09","flip":[]}</attribute>
     </xpath>
     <xpath expr="//section/div" position="before">
         <div class="o_we_shape o_web_editor_Blobs_09"/>
     </xpath>
+    <!-- Profile #1 -->
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//h4" position="replace" mode="inner">
+        <b>Tony Fred, Faculty Head of IT</b>
+    </xpath>
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//p" position="replace" mode="inner">
+        Tony received a degree in Electrical and Mechanical Engineering and a Ph D. degree in 1998 and 2004. After a post-doctoral experience  he joined the school as professor of mechatronics in 2006. In 2010, he became Head of IT.
+    </xpath>
+    <!-- Profile #2 -->
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div[2]//h4" position="replace" mode="inner">
+        <b>Mich Stark, IT Officer</b>
+    </xpath>
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div[2]//p" position="replace" mode="inner">
+        He is professor in the Institute of Mechanics, Materials and Civil Engineering since 2000. He lectures in mechanical drawing and mechanical design for undergraduate and graduate students. He is active in Problem and Project based learning. He is the promoter of 8 doctoral theses.
+    </xpath>
+    <!-- Profile #3 -->
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div[3]//h4" position="replace" mode="inner">
+        <b>Aline Turner, Law professor</b>
+    </xpath>
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div[3]//p" position="replace" mode="inner">
+        She has been practicing law at the French-speaking Brussels Bar since 2006. She has worked in various major law firms based in Brussels, as member and then head of their litigation/arbitration practice groups.
+    </xpath>
+    <!-- Profile #4 -->
+    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div[4]//h4" position="replace" mode="inner">
+        <b>Iris Joe, team leader professor</b>
+    </xpath>
+</template>
 
-    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//h4" position="replace">
-        <h5>Tony Fred, <small><b>Faculty Head of IT</b></small></h5>
+<!-- ======== BIG BOXES ======== -->
+<template id="s_color_blocks_2" inherit_id="website.s_color_blocks_2" name="Be Wise s_color_blocks_2">
+    <!-- Left column -->
+    <xpath expr="//div[hasclass('col-lg-6')]" position="attributes">
+        <attribute name="class" add="oe_img_bg o_bg_img_center o_cc4" remove="o_cc3" separator=" "/>
+        <attribute name="style">background-image: url('/web/image/website.s_media_list_default_image_1');</attribute>
+    </xpath>
+    <!-- Left column - Filter -->
+    <xpath expr="//div[hasclass('col-lg-6')]//i" position="before">
+        <div class="o_we_bg_filter" style="background-color: rgba(180, 144, 79, 0.8) !important;"/>
```

Comment:
Sure that filter work with all palette colors?

Reply:
Indeed, the color doesn't change. So I have to use only black/white filter classes. I'll change that.

Reply:
:+1: